devel/release.md

```diff
@@ -4,6 +4,24 @@ The release process of the python client involves creating (or updating) a relea
 branch, updating release tags, and creating distribution packages and uploading them to
 pypi.
 
+## Hot issues
+
+There are some hot issues with the client generation that require manual fixes.
+***The steps in this section should be performed after you finished the section "Update release tags".***
+
+1. Restore custom object patch hehavior. You should apply [this commit](https://github.com/kubernetes-client/python/pull/995/commits/9959273625b999ae9a8f0679c4def2ee7d699ede)
+to ensure custom object patch behavior is backwards compatible. For more
+details, see [#866](https://github.com/kubernetes-client/python/issues/866) and
+[#959](https://github.com/kubernetes-client/python/pull/959).
+
+2. Add alias package kubernetes.client.apis with deprecation warning. You need
+to add [this file](https://github.com/kubernetes-client/python/blob/0976d59d6ff206f2f428cabc7a6b7b1144843b2a/kubernetes/client/apis/__init__.py)
+under `kubernetes/client/apis/` to ensure the package is backwards compatible.
```

Comment:
Can we ask them to cherry-pick the commit instead or it was multiple commits?

Reply:
It could be done either by cherry-picking the commit or by copy-pasting the file.